ui/ducks/ramps/types.ts

```diff
@@ -0,0 +1,6 @@
+export type AggregatorNetwork = {
+  active: boolean;
+  chainId: number;
+  chainName: string;
+  nativeTokenSupported: boolean;
+};
```

Comment:
I mentioned this for the mocks but it is also related to the initial state and this type in general, it must have the `shortName: string;` property.

Reply:
https://github.com/MetaMask/metamask-extension/commit/988e98509c129d77fb4127cbed2f94b974ee4718